common/flatpak-installation.c

```diff
@@ -1655,6 +1655,31 @@ flatpak_installation_get_config (FlatpakInstallation *self,
   return flatpak_dir_get_config (dir, key, error);
 }
 
+/**
+ * flatpak_installation_get_default_locales:
+ * @self: a #FlatpakInstallation
+ * @error: return location for a #GError
+ *
+ * Get the default languages and locales used by frontends like
+ * GNOME Software to decide which language-specific apps to display.
```

Comment:
You could mention here that Flatpak itself only uses the language part of each when deciding which subpaths of locale extensions to download.

Reply:
Or just say "Like flatpak_installation_get_default_languages() but includes region information (e.g. en_US rather than en) which may be included in the xa.extra-languages configuration."